web/components/feed/feed-comments.tsx

```diff
@@ -180,20 +192,26 @@ export function FeedComment(props: {
           {onReplyClick && (
             <button
               className="font-bold hover:underline"
-              onClick={onReplyClick}
+              onClick={() => onReplyClick?.(comment)}
```

Comment:
`onReplyClick` should be defined here

Reply:
Can you clarify what you think should be changed here?

Reply:
Sure — you don't need the `?` in `onReplyClick?.(comment)` since `onReplyClick` is truthy in the `&&` condition above.